### DISCUSSION TITLE
Welcome to thingino-firmware Discussions!

### DISCUSSION THREAD

**Discussion by themactep** [themactep/thingino-firmware#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by grumpster3** [comment]
Do you have a T23 
Cinnado D1 firmware for the Smartsens SC2226 sensor?
I flashed the downloaded file and there is PTZ but no image.

**Reply by themactep** [reply]
Run sensor-info on the camera, confirm the sensor model.

**Reply by Buzzliteyear** [reply]
Is it possible to change the recorded video resolution thats captured on the SD card? 
and can i do it easily ?
